### PR TITLE
Task 31

### PR DESCRIPTION
**Pull request by RanimHelamiah**

### CODE REVIEW

File: lib/organisms/components/scalables/scalable1/index.tsx
Comment (AMhaish) [inline lib/organisms/components/scalables/scalable1/index.tsx:1]: I think we can consider this one as feature list too

File: lib/organisms/components/beta/beta1/index.tsx
Comment (AMhaish) [inline lib/organisms/components/beta/beta1/index.tsx:1]: I think we can consider this as a feature list

File: lib/organisms/components/sliders/slider3/index.tsx [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+const FeatureSlider = ({
+  row,
+  title,
+  description,
+  sectionSubTitle,
+  model,
+  isAR,
+  data,
+}: FeatureSliderProps) => {
+  let images = model.children.map((item, index) => {
```
Comment (AMhaish) [inline lib/organisms/components/sliders/slider3/index.tsx:64]:
```suggestion
  const images = model.children.map((item, index) => {
```

File: lib/organisms/components/transactions/transaction1/index.tsx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+  sectionSubTitle,
+  cardArea,
+  featureTitleStyle,
+  featureDescriptionStyle,
+  model,
+  isAR,
+  data,
+}: TransactionsHistoryProps) => {
+  return (
+    <TransactionsWrapper id="transactions">
```
Comment (AMhaish) [inline lib/organisms/components/transactions/transaction1/index.tsx:47]:
```suggestion
    <TransactionsWrapper id={model.name}>
```

File: lib/organisms/components/sliders/slider3/index.tsx [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+      map[obj.Key] = obj.Value;
+      return map;
+    },
+    {});
+    return {
+      original: imageMap.image,
+    };
+  });
+  return (
+    <FeatureSliderWrapper id="featureslider">
```
Comment (AMhaish) [inline lib/organisms/components/sliders/slider3/index.tsx:78]:
```suggestion
    <FeatureSliderWrapper id={model.name}>
```

File: lib/organisms/components/scalables/scalable1/index.tsx [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+  sectionSubTitle,
+  cardArea,
+  featureTitleStyle,
+  featureDescriptionStyle,
+  model,
+  isAR,
+  data,
+}: ScalableHistoryProps) => {
+  return (
+    <ScalableWrapper id="scalable" >
```
Comment (AMhaish) [inline lib/organisms/components/scalables/scalable1/index.tsx:48]:
```suggestion
    <ScalableWrapper id={model.name}>
    
```

I think we need to fix that for all sections of this template

File: lib/containers/Crypto/Transaction/index.js
Comment (AMhaish) [inline lib/containers/Crypto/Transaction/index.js:1]: I think also this one can be considered as feature list

In common I consider anything that contains icons with text as a feature list, it's like the owner of the website wants to list his product features
But every time this section shows the feature list in different ways

When a list contains images, then it should be a portfolio, not a feature list
When these images inside a slider with any type of animation, I think we can consider it as a slider

If you have any ideas feel free to let me know, I am just trying to think about the one who wants to build a website, and how he can find the right section for him quickly

File: lib/organisms/components/scalables/scalable1/index.tsx
Comment (RanimHelamiah) [reply lib/organisms/components/scalables/scalable1/index.tsx:1]: done

File: lib/organisms/components/beta/beta1/index.tsx
Comment (RanimHelamiah) [reply lib/organisms/components/beta/beta1/index.tsx:1]: done

File: lib/organisms/components/sliders/slider3/index.tsx (hunk 1 above)
Comment (RanimHelamiah) [reply lib/organisms/components/sliders/slider3/index.tsx:64]: done

File: lib/organisms/components/transactions/transaction1/index.tsx (hunk 2 above)
Comment (RanimHelamiah) [reply lib/organisms/components/transactions/transaction1/index.tsx:47]: done

File: lib/organisms/components/sliders/slider3/index.tsx (hunk 3 above)
Comment (RanimHelamiah) [reply lib/organisms/components/sliders/slider3/index.tsx:78]: done

File: lib/organisms/components/scalables/scalable1/index.tsx (hunk 4 above)
Comment (RanimHelamiah) [reply lib/organisms/components/scalables/scalable1/index.tsx:48]: done

**Comment by AMhaish** [comment]
@RanimHelamiah we have a conflict on this branch. can you fix it so we can merge it

**Pull request merged**